lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
It was always returning `id` by default and returned the whole inserted instance to the user (I think dao just appends back the `id`). Now it returns `id` and `_rev` and the user still gets back the whole instance.